drivers/filesystems/ntfs/mft.c

```diff
@@ -249,19 +249,19 @@ ULONGLONG
 AttributeAllocatedLength(PNTFS_ATTR_RECORD AttrRecord)
 {
     if (AttrRecord->IsNonResident)
-        return AttrRecord->NonResident.AllocatedSize;
+        return (ULONGLONG)AttrRecord->NonResident.AllocatedSize; // From LONGLONG.
```

Comment:
Are you editing in notepad, or why else do you feel the need to annotate what type it is being cast from?